sdk/tables/azure-data-tables/azure/data/tables/aio/_base_client_async.py

```diff
@@ -27,6 +28,7 @@
     HttpLoggingPolicy,
     UserAgentPolicy,
     ProxyPolicy,
+    AzureSasCredentialPolicy
```

Comment:
These are the same types for sync/async right?
Pretty sure we had no async-specific types for this auth...

Reply:
Yes you are correct